website/addons/wiki/static/WikiEditor.js

```diff
@@ -3,56 +3,143 @@
 * leaving the page with unsaved changes.
 */
 'use strict';
-// TODO: Use commonjs+webpack to load pagedown modules
 var ko = require('knockout');
 var $ = require('jquery');
 var $osf = require('osfHelpers');
 var Raven = require('raven-js');
 require('bootstrap-editable');
+var Markdown = require('pagedown-ace-converter');
+Markdown.getSanitizingConverter = require('pagedown-ace-sanitizer').getSanitizingConverter;
+require('imports?Markdown=pagedown-ace-converter!pagedown-ace-editor');
+
+var editor;
+
+ko.bindingHandlers.ace = {
+    init: function(element, valueAccessor) {
```

Comment:
Is `valueAccessor()` required to be an KO observable? If so, please document this.